Makefile

```diff
@@ -448,10 +448,10 @@ deploy-dev-infra: create-hub-cluster
 ########
 
 test:
-	$(MAKE) _run_subsystem_test AUTH_TYPE=rhsso ENABLE_ORG_TENANCY=true ENABLE_ORG_BASED_FEATURE_GATES=true
+	$(MAKE) _run_subsystem_test AUTH_TYPE=rhsso ENABLE_ORG_TENANCY=true ENABLE_ORG_BASED_FEATURE_GATES=true TEST="$(or $(TEST),'github.com/openshift/assisted-service/subsystem')"
 
 test-kube-api:
-	$(MAKE) _run_subsystem_test AUTH_TYPE=local FOCUS="$(or ${FOCUS},kube-api)"
+	$(MAKE) _run_subsystem_test AUTH_TYPE=local TEST="$(or $(TEST),'github.com/openshift/assisted-service/subsystem/kubeapi')"
```

Comment:
No way this is running some code that is not local right ? e.g remote `github.com/openshift/assisted-service`

Reply:
It's literally the name of the namespace 

You can see here in go list...

```
paulmaidment@fedora ~/projects/workspaces/1/assisted-service/subsystem (MGMT-19258)$ go list ./...
github.com/openshift/assisted-service/subsystem
github.com/openshift/assisted-service/subsystem/kubeapi
github.com/openshift/assisted-service/subsystem/utils_test
```

It's the pattern that the package name is fully qualified with the module URI from go mod 

```
module github.com/openshift/assisted-service
```

Always easy to verify in any case, just change one test and watch it fail. Which I have already seen.